_candidates_en/YoshitakaOkuda_3.md

```diff
@@ -0,0 +1,22 @@
+---
+name: Yoshitaka Okuda
+title: "Reactive Messaging Patterns for Systems Integrations"
+length: 40
+audience: Intermediate
+language: Japanese
+twitter: https://twitter.com/yoskhdia
+github: https://github.com/yoskhdia
+icon: https://pbs.twimg.com/profile_images/814301502516826112/tkeIfq1H_400x400.jpg
+organization: Supership Inc.
+tags:
+  - Best Practices
+  - Software Design and Architecture
+  - Microservices
+suggestions:
+  - People who want to read Reactive Messaging Patterns
+  - People who are interested in systems integrations using Akka
+  - People who want to know how to apply Akka
+---
+The book "Reactive Messaging Patterns with the Actor Model（Addison-Wesley Professional）"（RMP） was written by Vaughn Vernon, the author of "Implementing Domain-Driven Design（Addison-Wesley Professional）". This book is about how we can implement "Enterprise Integration Patterns"（EIP） using Akka.
```

Comment:
+The book "Reactive Messaging Patterns with the Actor Model（Addison-Wesley Professional）"（RMP）was written by Vaughn Vernon, the author of "Implementing Domain-Driven Design（Addison-Wesley Professional）". This book is about how we can implement "Enterprise Integration Patterns"（EIP） using Akka.